ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/transaction/jts/jca/TransactionImporterImple.java

```diff
@@ -10,6 +10,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicReference;
 
+import com.arjuna.ats.arjuna.coordinator.TxControl;
```

Comment:
I think this looks like it should be reverted too?

Reply:
I don't understand how I'm keeping miss these oversights :facepalm: Sorry for wasting your time :face_with_peeking_eye:

Reply:
No problem, happy to help!

Reply:
Done. Hopefully, this one is the last one.